TypeCobol.Test/Parser/Programs/Cobol85/MissingSpaceAfterDotDecimalPointIsComma.rdz.cbl

```diff
@@ -15,7 +15,7 @@
        SOURCE-COMPUTER.IBM-370.
       *Warning: A blank was missing before character "I" in column 24.
       *         A blank was assumed.
-       OBJECT-COMPUTER.IBMCOBOLII.
+                OBJECT-COMPUTER.IBMCOBOLII.
```

Comment:
OBJECT-COMPUTER must starts in Area A. I forgot that in the issue.
Can you update ObjectComputerParagraph class ?

Reply:
@laabidiHend Can you check this comment.